selfdrive/car/honda/carstate.py

```diff
@@ -206,6 +206,10 @@ def update(self, cp, cp_cam, cp_body):
         ret.cruiseState.standstill = acc_hud["CRUISE_SPEED"] == 252.
 
         conversion = get_cruise_speed_conversion(self.CP.carFingerprint, self.is_metric)
+        if self.CP.carFingerprint == CAR.HONDA_ODYSSEY_BOSCH:
+          conversion = CV.MPH_TO_MS if acc_hud['IMPERIAL_UNIT'] else CV.KPH_TO_MS
+        else:
+          conversion = get_cruise_speed_conversion(self.CP.carFingerprint, self.is_metric)
```

Comment:
this is the same as the line above. why can't `get_cruise_speed_conversion` handle this?